docs/psmdb-pro.md

```diff
@@ -14,7 +12,9 @@ Find the list of solutions available in Percona Server for MongoDB Pro builds:
 
 | Name                                | Version added | Description  | 
 | ----------------------------------- | ------------- | -------------
-| [FIPS support ](fips.md)| [7.0.4-2](https://docs.percona.com/percona-server-for-mongodb/release_notes/7.0.4-2.html) | FIPS mode provides a way to use FIPS-compliant encryption and run the Percona Server for MongoDB with the FIPS-140 certified library for OpenSSL. This helps customers meet minimum security requirements for cryptographic modules and testing in both hardware and software. |
+| [FIPS support ](fips.md)| 8.0.4-1 | FIPS mode provides a way to use FIPS-compliant encryption and run the Percona Server for MongoDB with the FIPS-140 certified library for OpenSSL. This helps customers meet minimum security requirements for cryptographic modules and testing in both hardware and software. |
+| Binaries with debug symbols | 8.0.6-3](release_notes/8.0.6-3.md) | By including debug symbols in the binary, Percona Server for MongoDB enables deeper integration with monitoring agent-based solutions. These agents can instrument the binary at runtime, providing more detailed telemetry data, such as performance metrics, error tracking, and function-level diagnostics. This enhanced observability allows for better monitoring of system health, faster identification of issues, and more granular insights into how the application performs in production environments.<br> Including this information empowers teams to respond proactively to performance bottlenecks, optimize resource allocation, and improve the overall stability of the application with real-time insights. |
```

Comment:
Another broken link to non-existent release notes?

Reply:
Fixed